frontend/app/src/composables/message-handling.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
We need to figure out a way to break this file into smaller files @rotki/frontend.

In general go through the code find out where we disable the max-lines rule and start splitting.